lms/djangoapps/certificates/admin.py

```diff
@@ -2,7 +2,8 @@
 django admin pages for certificates models
 """
 from django.contrib import admin
-from certificates.models import CertificateGenerationConfiguration
+from certificates.models import CertificateGenerationConfiguration, CertificateHtmlViewConfiguration
 
 
 admin.site.register(CertificateGenerationConfiguration)
+admin.site.register(CertificateHtmlViewConfiguration)
```

Comment:
There is a `ConfigurationModelAdmin` that provides a better audit trail for configuration changes.